doc/advanced/content/pcl_style_guide.rst

```diff
@@ -317,64 +317,28 @@ indentation level of the class body and members affected by these qualifiers are
 2.6. Automatic code formatting
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The following set of rules can be automatically used by various different IDEs,
-editors, etc.
+We currently use clang-format as the tool for auto-formatting our C++ code. The style files which were previously distributed should now be considered deprecated.
 
-2.6.1. Emacs
-""""""""""""
+The style rules mentioned in this document are enforced via `PCL's .clang-format file
+<https://github.com/PointCloudLibrary/pcl/blob/master/.clang-format>`_.
 
-You can use the following `PCL C/C++ style file
-<https://raw.githubusercontent.com/PointCloudLibrary/pcl/master/doc/advanced/content/files/pcl-c-style.el>`_,
-download it to some known location and then:
+For the integration of clang-format with various text editors and IDE's, refer to this `page
+<https://clang.llvm.org/docs/ClangFormat.html>`_.
 
-* open .emacs 
-* add the following before any C/C++ custom hooks
+Details about the style options used can be found `here
+<https://clang.llvm.org/docs/ClangFormatStyleOptions.html>`_.
 
-::
+2.6.1. Script usage
+"""""""""""""""""""
 
-   (load-file "/location/to/pcl-c-style.el")
-   (add-hook 'c-mode-common-hook 'pcl-set-c-style)
+PCL also has a supporting format script which runs the clang-format on the whitelisted directories.
```

Comment:
It's not a script, rather a build target:

PCL also creates a build target 'format' to format the whitelisted directories using clang-format.

Reply:
I was referring to `.dev/format.sh`, but your suggestion is much better :+1: 
Done.